sdk/core/azure_core_amqp/Cargo.toml

```
@@ -46,6 +47,7 @@ fe2o3-amqp = [
 ]
 cplusplus = []
 test_e2e = []
+cpp-repo = ["uuid"]
```

New guideline: all features must use underscores. Also, to disambiguate, I've been prefacing dependencies with "dep":

```suggestion
cpp_repo = ["dep:uuid"]
```